src/plugins/julia/templates/lr.template.jl

```diff
@@ -0,0 +1,234 @@
+module SyntaxParser
+
+#=
+  LR parser generated by the Syntax tool.
+
+  https://www.npmjs.com/package/syntax-cli
+
+    npm install -g syntax-cli
+
+    syntax-cli --help
+
+  To regenerate run:
+
+    syntax-cli \
+      --grammar ~/path-to-grammar-file \
+      --mode <parsing-mode> \
+      --output ~/ParserClassName.jl
+=#
+
+# --------------------------------------------------------------
+# Exports: The parse method, if concerned about disambiguating with other
+# parse methods in your use case, use 'import' instead of 'using'
+export parse
+
+# --------------------------------------------------------------
+# Shared includes
+using DataStructures
+
+# Constants and globals
+const EOF = "\$"
+yytext = ""
```

Comment:
Per discussion, see if we can remove this globals, and make them per parser/tokenizer data structure (as it's done in [other plugins](https://github.com/DmitrySoshnikov/syntax/blob/master/src/plugins/rust/rust-parser-generator-trait.js#L349)).  We can do this as an optimization in the follow up PRs.

Reply:
I will be working on this in the near future